modules/http-server/package.json

```diff
@@ -3,31 +3,33 @@
   "version": "1.0.0",
   "description": "",
   "repository": "phenyl-js/phenyl",
-  "main": "dist/index.js",
-  "types": "dist/index.d.js",
-  "scripts": {
-    "build": "upbin tsc --declaration",
-    "type-check": "upbin tsc --noEmit"
-  },
+  "license": "Apache-2.0",
+  "author": [
+    "Leko <leko.noor@gmail.com>",
+    "Shin Suzuki <shinout310@gmail.com>",
+    "imoans <naomins0101@gmail.com>"
+  ],
   "files": [
     "src",
-    "dist",
+    "lib",
     "package.json",
```

Comment:
`src` and `package.json` are unnecessary as they are already included by defautl.
please remove them from `files` :pray: